Dockerfile

```diff
@@ -31,6 +31,10 @@ ENV JAVA_HOME /usr/lib/jvm/default-java
 RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.10.1/bazelisk-linux-amd64 \
     -O /usr/local/bin/bazelisk && chmod a+x /usr/local/bin/bazelisk
 
+# Set up buildifier
```

Comment:
I think we will need to update the image the rbe uses. You could do it in a follow up CL. https://github.com/google-research/raksha/blob/6f701da6e7ebd40aff235f3b0c6531485014187c/gcb/rbe/configs/path/config/BUILD#L45

Related issue #226

This still caught the error in presubmit because we don't use RBE for buildifer?

Reply:
Updated the build image.

Reply:
I think we do use RBE for buildifier. But I think it caught the error in presubmit because building the image had a cache miss upon trying to fetch the result of the buildifier line and built the image from scratch from there. While still correct, that is slow. Upon updating the hash, it does appear to finish building the image much more quickly.